src/codemagic/apple/resources/build.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from datetime import datetime
+
+from .enums import BuildProcessingState
+from .resource import Resource
+from .resource import DictSerializable
+from .resource import Relationship
+
+
+class Build(Resource):
+    """
+    https://developer.apple.com/documentation/appstoreconnectapi/build
+    """
+
+    @dataclass
+    class Attributes(Resource.Attributes):
+        expired: bool
+        iconAssetToken: ImageAsset
+        minOsVersion: str
+        processingState: BuildProcessingState
+        version: str
+        usesNonExemptEncryption: bool
+        uploadedDate: datetime
+        expirationDate: datetime
+
+        def __post_init__(self):
+            if isinstance(self.processingState, str):
+                self.processingState = BuildProcessingState(self.processingState)
+            if isinstance(self.uploadedDate, str):
+                self.uploadedDate = Resource.from_iso_8601(self.uploadedDate)
+            if isinstance(self.expirationDate, str):
+                self.expirationDate = Resource.from_iso_8601(self.expirationDate)
+
+    @dataclass
+    class Relationships(Resource.Relationships):
+        app: Relationship
+        appEncryptionDeclaration: Relationship
+        individualTesters: Relationship
+        preReleaseVersion: Relationship
+        betaBuildLocalizations: Relationship
+        buildBetaDetail: Relationship
+        betaAppReviewSubmission: Relationship
+        appStoreVersion: Relationship
+        icons: Relationship
+
+        betaGroups: Relationship
+        perfPowerMetrics: Relationship
+        diagnosticSignatures: Relationship
+
+@dataclass
+class ImageAsset(DictSerializable):
```

Comment:
keeping it to not list it as a separate resource, it's only used in this class

Reply:
fixed [here](https://github.com/codemagic-ci-cd/cli-tools/pull/67/commits/71f2ca10806aa46be85d0feca84182780939f3d7#diff-51b8fb81410afec451e8bc989ee6a2cd03ab8b4f5df4f2423b4058c949bc2bd3R10)